src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Model/EncryptionProtectorType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
Make sure you add copyright header to all new cs files.

Reply:
Addressed